README.md

```diff
@@ -113,12 +113,12 @@ print(
 )
 
 # Run pagerank and ask for the top ranked node
-top_node = algo.pagerank(graph).top_k(1)
+top_node = algo.pagerank(graph).top_k(5).max_item()
```

Comment:
just max_item() should do?

Reply:
It does - I just wanted to show off the other APIs